pkg/graph/resolvers/plan_discussion.go

```diff
@@ -174,13 +196,30 @@ func CreateDiscussionReply(
 	if err != nil {
 		return nil, err
 	}
+	err = UpdateTaggedHTMLMentionsAndRawContent(ctx, store, &discussionReply.Content, getAccountInformation)
 
-	result, err := store.DiscussionReplyCreate(logger, discussionReply)
-	return result, err
+	if err != nil {
+		logger.Info("not all mentions were able to be updated")
+		//TODO: do we need to stop execution here? Should we silently continue instead? Should we filter out any bad tags?
+		// return nil, err
+	}
+
+	reply, err := store.DiscussionReplyCreate(logger, discussionReply)
+	if err != nil {
+		return reply, err
+	}
+	//TODO: should we put this in a transaction?
+	tags, err := TagCollectionCreate(logger, store, principal, "content", "discussion_reply", reply.ID, discussionReply.Content.Mentions)
+	if err != nil {
+		return reply, err
+	}
+	reply.Content.Tags = tags
+	return reply, err
 }
 
 // UpdateDiscussionReply implements resolver logic to update a Discussion reply object
 func UpdateDiscussionReply(logger *zap.Logger, id uuid.UUID, changes map[string]interface{}, principal authentication.Principal, store *storage.Store) (*models.DiscussionReply, error) {
+	// THIS IS NOT USED by the front end. If it is ever used, make sure to handle tags
```

Comment:
Nit: Can you update these comments to be in the function comments using the `Deprecated:` prefix? That way they'll show up to anyone trying to call the functions

https://stackoverflow.com/a/36360323